lib/connect_proto/src/result.proto

```diff
@@ -28,10 +28,11 @@ message Result {
     STRING = 14;
     TIME = 15;
     STRUCTURED_NUMERIC = 16;
+    ADDRESS = 17;
   }
 
   enum AbnormalFlag {
-    NORMAL = 0;
+    NONE = 0;
```

Comment:
Doesnt `NORMAL` signify the same thing as `NONE` in this case? (Abnormal Flag = normal)?